art/attacks/evasion/boundary.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         num_trial: int = 25,
         sample_size: int = 20,
         init_size: int = 100,
-        min_epsilon: Optional[float] = None,
+        min_epsilon: float = 0,
```

Comment:
```suggestion
        min_epsilon: Union[float, int] = 0,
```

Reply:
In [PEP 484](https://www.python.org/dev/peps/pep-0484/#the-numeric-tower),
>when an argument is annotated as having type float, an argument of type int is acceptable

so I think only `float` is sufficient as in `epsilon` and `delta`.